index-server/ergvein-index-server.cabal

```diff
@@ -0,0 +1,117 @@
+name:                ergvein-index-server
+version:             0.1.0.0
+synopsis:            Ergvein wallet cryptocurrencies index server
+description:         See README.md
+category:            Cryptocurrency
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:
+    Ergvein.Index.Server.App
+    Ergvein.Index.Server
+    Ergvein.Index.Server.Server
+    Ergvein.Index.Server.Monad
+    Ergvein.Index.Server.Server.V1
+    Ergvein.Index.Server.DB.Schema
+    Ergvein.Index.Server.DB.Queries
+    Ergvein.Index.Server.DB.Monad
+    Ergvein.Index.Server.Config
+
+  build-depends:
+        base
```

Comment:
Need version bounds for most packages. Current major version is good enough for now. Without constraints on base hackage will not accept the package.